File: Courzelo-backend-LMS/Courzelo/src/main/java/org/example/courzelo/dto/responses/UserResponse.java
package org.example.courzelo.dto.responses;

import lombok.Builder;
import lombok.Data;
import org.example.courzelo.models.User;
import org.example.courzelo.models.UserProfile;

import java.util.List;

@Data
@Builder
public class UserResponse {
     private String id;
    String email;
    List<String> roles;
    UserProfileResponse profile;
    UserSecurityResponse security;
    UserEducationResponse education;
}

File: courzelo-dashboard-LMS-main/src/app/shared/models/user/UserResponse.ts
import {UserProfileResponse} from './UserProfileResponse';
import {UserSecurityResponse} from './UserSecurityResponse';
import {UserEducationResponse} from './UserEducationResponse';

export interface UserResponse {
    id?: string;
    email?: string;
    roles?: string[];
    profile?: UserProfileResponse;
    security?: UserSecurityResponse;
    education?: UserEducationResponse;
}
